readthedocs/projects/filters.py

```diff
@@ -0,0 +1,158 @@
+import logging
+
+from django.db.models import Count, F, Max
+from django.forms.widgets import HiddenInput
+from django.utils.translation import ugettext_lazy as _
+from django_filters import CharFilter, ChoiceFilter, FilterSet, OrderingFilter
+
+log = logging.getLogger(__name__)
+
+
+class SortOrderingFilter(OrderingFilter):
```

Comment:
I didn't find _where_ you use this custom filter.

Reply:
Hrm, then I might have busted something in all of the merges. Let me check